contessa/executor.py

```diff
@@ -67,14 +67,26 @@ def compose_where_time_filter(self, rule):
         only data that were updated/created/confirmed in last 30 days.
         :return: str, WHERE `time_filter` filter statement
         """
-        time_filter_column = rule.time_filter
-        if time_filter_column:
-            past = (datetime.now() - timedelta(days=30)).strftime(
-                "%Y-%m-%d %H:%M:%S UTC"
-            )
-            return f"""{time_filter_column} >= '{past}'::timestamp"""
-        else:
-            return ""
+        days = 30
+        filters = []
+        result = []
+        if rule.time_filter:
+            if isinstance(rule.time_filter, str):
+                filters.append({"column": rule.time_filter, "days": days})
+            elif isinstance(rule.time_filter, list):
+                for each in rule.time_filter:
+                    filters.append(
+                        {"column": each.get("column"), "days": each.get("days", days)}
```

Comment:
I would make `column` field required. for the days, lets keep 30 now to not break compatibility, but I would maybe consider that required.

Maybe it make sense to validate rules outside of `Executor`, in Normalizer e.g., what do you think ?

Reply:
To require _column_ is ok by me, I'll add it.
Central validation would be definitely good, normalizer is good place. I'll make separate issue for it.